rpc/constants.go

```diff
@@ -5,8 +5,6 @@ const MetadataServiceAccountNameKey = "x-orchard-service-account-name"
 //nolint:gosec // G101 check yields a false-positive here, this is not a hard-coded credential
 const MetadataServiceAccountTokenKey = "x-orchard-service-account-token"
 
-const MetadataWorkerNameKey = "x-orchard-worker-name"
-
-const MetadataWorkerUIDKey = "x-orchard-worker-uid"
+const MetadataWorkerKey = "x-orchard-worker"
```

Comment:
What do you think of keeping the `MetadataWorkerNameKey` since we will actually use just the name?

Reply:
See https://github.com/cirruslabs/orchard/pull/48/commits/d86079cf63abc3c670291e842f1294d6cae6039b.